sw/host/tests/rom/e2e_bootstrap_rma/gen_rma_token.py

```diff
@@ -0,0 +1,18 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+from Crypto.Hash import cSHAKE128
+
+TOKEN_LEN = 16
+# This hardcoded preimage was generated by `random.randbytes(16)`.
+TOKEN_PREIMAGE = b'\x15\x85\x03\xc8\xa9(\xb8\xb9\x0b\x0c4b\xdf\x97\xc3e'
```

Comment:
nit: I think randbytes is introduced in 3.9 but we use 3.8. Can we use something like `bytes([random.randrange(0, 256) for _ in range(6)])`?

Reply:
I think we're targeting 3.9, according to [third_party/python/deps.bzl](https://cs.opensource.google/opentitan/opentitan/+/master:third_party/python/deps.bzl;l=10;drc=98cfa22fb3f2455fa23abf656a8926b857175f86). Is that right?

Not that it matters for this test, but to set a good example, I suppose we should be using `secrets.token_bytes()`.

Reply:
oh, I wasn't aware of that change :) my opentitan container still has python3.8 installed locally. Either I am using an old one, or the container image doesn't use the same python version as our repo.